### PR TITLE
WIP PR for BXMSDOC-8484: Add a section about setting the priority of a task in the Business Central.

### PR DESCRIPTION
**Pull request by kaldesai**
**Epic:** https://issues.redhat.com/browse/BXMSDOC-8483
**Tracking JIRA:** https://issues.redhat.com/browse/BXMSDOC-8484

**Doc preview:** [III. Managing and monitoring business processes in Business Central](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-8484-task-priority-RHPAM/#setting-the-priority-of-task-MVEL-proc) 

**Doc impact:** Please check and verify section **29.6. Setting the priority of a task as an MVEL expression** of the shared document and let me know what you think.

### CODE REVIEW

File: doc-content/enterprise-only/admin-and-config/setting-the-priority-of-task-MVEL-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+= Setting the priority of a task as an MVEL expression
+
+You can set the priority of a task as an MVEL expression in {CENTRAL} from the *Task Inbox* page. Note that all users may not have permissions for setting priority of a task.
+
+.Prerequisites
+You have created a project in {CENTRAL}.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Track* -> *Task Inbox*.
+. On the *Task Inbox* page, click the task to open it.
```
Comment (csherrar) [inline doc-content/enterprise-only/admin-and-config/setting-the-priority-of-task-MVEL-proc.adoc:11]: [Style question] - might it be better to refer to the task UI element or perhaps rephrase to "To do this, do that":

s/click the task to open it.
/click the name of the task you want to open.
/click the task name to open the task.
/to open a specific task, click the task name.

**Review by csherrar — COMMENTED** [review]
Other than a few questions and suggestions, looks good to me, Kalyani!

**Review by LuboTerifaj — CHANGES_REQUESTED** [review]
Hello @kaldesai ,

[BAPL-2244](https://issues.redhat.com/browse/BAPL-2244) was actually about enabling expressions in Process Designer editor, not in Task inbox. It is not possible to set expressions in Task Inbox.

**Comment by kaldesai** [comment]
> Hello @kaldesai ,
> 
> [BAPL-2244](https://issues.redhat.com/browse/BAPL-2244) was actually about enabling expressions in Process Designer editor, not in Task inbox. It is not possible to set expressions in Task Inbox.

Hey @LuboTerifaj 
We confirmed the doc impact from Jamie on the [BAPL-2244](https://issues.redhat.com/browse/BAPL-2244) itself and also received an ack for the content written. Here: [BXMSDOC-8539](https://issues.redhat.com/browse/BXMSDOC-8539)

Can you please let us know what will be the correct location in the doc to include this then?
Doc: [Developing process services in Red Hat Process Automation Manager](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.12/html-single/developing_process_services_in_red_hat_process_automation_manager/index#viewing-task-variables-advanced-filters-in-bc-proc)

**Comment by LuboTerifaj** [comment]
Hello @kaldesai,

Unfortunately it is not possible to set expressions in Task Inbox. See my comment in [BXMSDOC-8541](https://issues.redhat.com/browse/BXMSDOC-8541).

Thank you!

**Comment by LuboTerifaj** [comment]
Hi @kaldesai , any updates? Thanks!

**Pull request closed without merging**